controller/planner_backlog_test.go

```diff
@@ -95,7 +94,7 @@ func (rest *TestPlannerBacklogREST) setupPlannerBacklogWorkItems() (testSpace *s
 			SpaceID:    testSpace.ID,
 			Path:       append(parentIteration.Path, parentIteration.ID),
 			State:      iteration.IterationStateStart,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c